Monika After Story/game/script-topics.rpy

```diff
@@ -4697,4 +4697,23 @@ label monika_omamori:
     m "But maybe with one of these, I can get just a bit closer to your world."
     m 1k "I could be your guardian deity, ehehe~"
     return    
+    
+    init 5 python:                                                  #Added one new topic
+          # List of keywords for this topic.
+ -        for key in ['smoke', 'cigarettes', 'cigarette', 'smoking']:
+ +        for key in ['smoke', 'cigarettes', 'cigarette', 'smoking', 'tobacco']:
+              monika_topics.setdefault(key,[])
+              monika_topics[key].append('monika_smoking')
+          monika_random_topics.append('monika_smoking') 
```

Comment:
The keyword system is being replaced (see #126 ). Your topic (aka `Event`) should follow the new framework documented in the linked issue. You can also find documentation on the `Event` class in `definitions`.